jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Handler.java

```diff
@@ -335,11 +336,7 @@ default List<Handler> getHandlers()
          */
         default void insertHandler(Singleton handler)
         {
-            Singleton tail = handler;
-            while (tail.getHandler() instanceof Wrapper)
-            {
-                tail = (Wrapper)tail.getHandler();
-            }
+            Singleton tail = Objects.requireNonNull(handler).getTail();
```

Comment:
Remove unnecessary `Objects.requireNonNull()`, adds nothing to a naked call to `handler.getTail()`.